listener/src/graphQLClient.ts

```diff
@@ -6,11 +6,12 @@ import { execute } from 'apollo-link';
 import { WebSocketLink } from 'apollo-link-ws';
```

Comment:
such a nit pick, but feel like we should split this file

Reply:
I'm going to keep it like this for now, but I'm thinking it should be made into it's own package for other clients to use